src/CreeDictionary/CreeDictionary/templates/CreeDictionary/components/linguistic-breakdown.html

```diff
@@ -35,8 +35,8 @@ <h3 class="linguistic-breakdown__stem"> {{ result.lemma_wordform.stem }} </h3>
           {% endif %}
 
           <ol class="linguistic-breakdown__list">
-            {% for tag in result.relevant_tags %}
-              <li><data value="{{ tag.value }}">{{ tag.in_plain_english }}</data></li>
+            {% for chunk in result.relabelled_fst_analysis %}
```

Comment:
I tried combinging the display mode here with the relabelled FST analysis (having the serialization contain relabellings for all display modes by default), and it was way more complicated than I thought it was. If you have any ideas how to do that here, rather than pulling `display_mode` down several levels of abstraction, please let me know!